src/package_io/input_parser.star

```diff
@@ -457,7 +457,9 @@ def default_grafana_params(registry):
     return {
         "image": registry.get(_registry.GRAFANA),
         "dashboard_sources": [
-            "github.com/ethereum-optimism/grafana-dashboards-public/resources"
+            "github.com/ethereum-optimism/grafana-dashboards-public/resources",
+            "github.com/op-rs/kona/docker/recipes/kona-node/grafana",
+            "github.com/paradigmxyz/reth/etc/grafana"
```

Comment:
Are you sure you want these as defaults?

Reply:
Yeah; any harm in doing so?